Source/Core/DolphinQt2/QtUtils/SignalDaemon.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/QtUtils/SignalDaemon.h"
+
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#include <QSocketNotifier>
+
+int SignalDaemon::s_sigterm_fd[2];
+
+static const std::string interrupt_message =
```

Comment:
Note: this won't do what you want because QObject::tr will be called before main() and before Dolphin's translation code even gets called.

Also, I'm not sure if this is safe... a quick search seems to indicate that [this can cause crashes](http://lists.qt-project.org/pipermail/interest/2015-November/019639.html) because the metaobject system hasn't been initialised yet.

I'd just not bother with translating this and make this a dumb `constexpr char[]`.